dirsrvtests/tests/suites/betxns/betxn_test.py

```diff
@@ -29,6 +29,13 @@
 log = logging.getLogger(__name__)
 USER_PASSWORD = 'password'
 
+def retry_if_busy(f):
+    while True:
```

Comment:
Do you think it worth adding a max retry ? Just to prevent an infinite loop of the test if similar problem popup again ?

Reply:
Probably a good idea (I needed to add such limit in my testcase because first patch attempt was always returning LDAP_BUSY).
Note: the change is pretty simple:
 I replaced the `while True:`  by: `for x in range(1000):` and 
```
log.error('Too many retries')
assert False
```
 after the loop

Reply:
Thanks for the feedback. I've amended the commit with the recommended changes.